src/eduid_common/api/session.py

```diff
@@ -271,7 +271,7 @@ def open_session(self, app, request):
                 base_session = self.manager.get_session(token=token)
                 session = Session(app, base_session, new=False)
                 current_app.logger.debug('Loaded existing session {}'.format(session))
-            except KeyError:
+            except (KeyError, ValueError):
                 # I (@john) approve of commit 974836137eb83fd163abe4d83b12a9c6033c127e
```

Comment:
maybe you can clean up this comment now while we're changing these lines of code? =)